static/js/mediastream/peercall.js

```diff
@@ -130,6 +143,10 @@ define(['jquery', 'underscore', 'mediastream/utils', 'mediastream/peerconnection
 
 		console.error("Failed to create answer/offer", event);
 
+		// Even though the Offer/Answer could not be created, we now allow
+		// to create own Offers to perform renegotiation again.
+		this.peerconnection.readyForRenegotiation = true;
```

Comment:
This should probably be a function, rather than directly setting an external member.
 Add a line note